packages/geo/src/lib/query/shared/query.interfaces.ts

```diff
@@ -17,7 +17,7 @@ export interface QueryableDataSourceOptions extends DataSourceOptions {
   queryable?: boolean;
   queryFormat?: QueryFormat;
   queryTitle?: string;
-  queryUrl?: string;
+  queryUrl?: QueryUrlData[];
```

Comment:
I think queryUrl refer to a single url...

If you now refer to an array, I think queryUrls is more appropriate